osu.Game/Screens/Play/HUD/UnstableRateCounter.cs

```diff
@@ -39,6 +47,22 @@ public UnstableRateCounter()
         [BackgroundDependencyLoader]
         private void load(OsuColour colours)
         {
+            if (Convert.Value)
+            {
+                var rateAdjustMod = scoreProcessor.Mods.Value
+                    .Where(sp => sp is ModRateAdjust)
+                    .FirstOrDefault() as ModRateAdjust;
```

Comment:
Also this isn't considering `ModTimeRamp` (tricky, but somewhat doable) or `ModAdaptiveSpeed` (I don't even know where you _begin_ with that one).

Reply:
I decided not to address those mods (adaptive speed in specific) due to their dynamic nature, and inability to convert the unstable rate from a result screen alone. Wind Up is most likely doable, but I didn't want to be overcomplicating the implementation for something I wasn't even sure was gonna be accepted in the first place